air/src/proof/ood_frame.rs

```diff
@@ -45,49 +39,53 @@ impl OodFrame {
     // UPDATERS
     // --------------------------------------------------------------------------------------------
 
-    /// Updates the trace state portion of this out-of-domain frame. This also returns a compacted
-    /// version of the out-of-domain frame (including the Lagrange kernel frame, if any) with the
-    /// rows interleaved. This is done so that reseeding of the random coin needs to be done only
-    /// once as opposed to once per each row.
+    /// Updates the trace state portion of this out-of-domain frame, and returns the hash of the
+    /// trace states.
+    ///
+    /// The out-of-domain frame is stored as one vector of interleaved values, one from the current
+    /// row and the other from the next row. Given the input frame
+    ///
+    ///    +-------+-------+-------+-------+-------+-------+-------+-------+ |  a1   |   a2  |  ...
+    ///    |   an  |  c1   |  c2   |  ...  |  cm   |
+    ///    +-------+-------+-------+-------+-------+-------+-------+-------+ |  b1   |   b2  |  ...
+    ///    |   bn  |  d1   |  d2   |  ...  |  dm   |
+    ///    +-------+-------+-------+-------+-------+-------+-------+-------+
```

Comment:
I think the formatting here is a bit off.

Reply:
oops, fixed